test/sly-package-fu-tests.el

```diff
@@ -0,0 +1,54 @@
+;; -*- lexical-binding: t; -*-
+(require 'sly-package-fu "contrib/sly-package-fu")
+(require 'sly-tests "lib/sly-tests")
+
+(def-sly-test sly-package-fu-sly-import (initial-defpackage final-defpackage symbol)
+              "Check if importing `import` on `initial-defpackage` results in `final-defpackage."
+              '(("(defpackage :foo
```

Comment:
Use `\n` to put this in one line.  Or find some way to use lisp forms, instead of strings representing lisp forms, using quasiquote, for example.

Reply:
I followed the pattern used in some other tests, for instance sly-fontifying-fu-tests. The problem with putting it all on one line, is that it'll be hard to verify that every thing is indented appropriately. (Alternatively, I can update the tests to disregard indentation, i.e. convert multiple spaces to a single space. It'll test a few less things, but indentitation is not a priority for this test anyway.)

Reply:
Make fixtures in simple lisp forms.  Other tests are not necessarily good examples.  Indentation is not relevant at all for these tests.

Reply:
Something like this (untested)

```lisp
(def-sly-test foo-test (initial expected symbol)
  "Foo the foo"
  '(((defpackage :foo bla) (defpackage :foo bla-with-the-airplane) cl:airplane)
    ((defpackage :another-fixture ...) (defpackage :another-fixture now-changed ...) cl:airplane))
  (with-temp-buffer
    (lisp-mode)
    (insert (pp-to-string initial))
    (do-things-with-package-fu)
    (should (equal (read (buffer-string)) ; not sure read works like this in elisp, look it up
                   expected)))
```

Reply:
@joaotavora I just gave this a try, and sadly.. the current version of sly-package-fu does not handle the entire expression being on a single line, since it uses regexps to find where to insert things. So at least the input state will need to be multiple lines. 

Are you okay with keeping it this way for ease of modification? (As mentioned before, using \n will make it really hard to edit.